### DISCUSSION TITLE
How do I move my self-hosted Kimai 2 database to a MySQL server?

### DISCUSSION THREAD

**Discussion by stevesobol** [kimai/kimai#3309] [Questions & Answers] [answered]
Hello,
My Kimai site is going to remain on the same web server (for now), but I need to move the database to a new MySQL server. The website and database are on different VMs.

I literally just need to have Kimai connect to the new MySQL server - the login credentials won't change.

I know how to physically move the database. But I haven't figured out how to tell Kimai to connect to the new server. Help, please!

**Comment by kevinpapst** [comment, marked as answer]
Can you figure out how to connect Kimai to a database when reading the (installation) docs?

**Comment by stevesobol** [comment]
The docs say to run bin/console kimai:install, but kimai:install is prompting me for the details for a new database.

At least I think that's what it's doing. I must be missing something.
 

Friday, May 13, 2022, 12:57:08 AM, you wrote:

**Reply by kevinpapst** [reply]
It's all in the docs:
https://www.kimai.org/documentation/installation.html
https://www.kimai.org/documentation/local-yaml.html#env
https://www.kimai.org/documentation/backups.html

**Reply by stevesobol** [reply]
Well, now I feel silly.

I couldn't figure out where my database settings were. They're in `.env`, in the web root. I searched for the hostname of my old MySQL server... the problem is, grep doesn't search dotfiles by default.

All good. Thanks
